tendrl/commons/message.py

```diff
@@ -3,8 +3,14 @@
 from inspect import getframeinfo
 from inspect import stack
 import json
+import sys
 from tendrl.commons.utils.time_utils import now
+sys.path.remove('/usr/lib64/collectd')
 import uuid
+sys.path.append('/usr/lib64/collectd')
```

Comment:
Are you sure collectd python modules are only found in "/usr/lib64/collectd"  and not at places like "/usr/lib/collectd" or any random places?